src/locale/locales/pt-BR/messages.po

```diff
@@ -939,11 +939,11 @@ msgstr "Bluesky é melhor com amigos!"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:282
 msgid "Bluesky will choose a set of recommended accounts from people in your network."
-msgstr "O Bluesky escolherá um conjunto de contas recomendadas de pessoas em sua rede."
+msgstr "O Bluesky escolherá um conjunto de contas recomendadas das pessoas em sua rede."
```

Comment:
I did this one on my earlier PR but I think maybe it'd be better if we used the preposition "entre as" instead of "das"?

Reply:
Or "dentre as"

Reply:
This adjustment was in #5090 next fixed commit.